### PR TITLE
feat: added s3 download support

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: src/exportImage.py [hunk 1]
```diff
@@ -67,9 +67,17 @@ def export(self, bbox, filename, url, taskid, directoryName, max_zoom):
...
-                        self.upload_to_s3(filename, directoryName, output_format, full_path)
-                        self.delete_local_directory(directoryName)
-                    self.__helper.save_update(
+                        try:
+                            s3_download_url = self.upload_to_s3(filename, directoryName, output_format, full_path)
+                        except Exception as e:
+                            raise e
+                        finally:
+                            self.delete_local_directory(directoryName)
+                            self.__helper.save_update(
```
Comment (asafMasa) [inline src/exportImage.py]: why? if failed to load to s3 than the process was NOT completed successfully

File: src/exportImage.py [hunk 2]
```diff
@@ -90,11 +98,17 @@ def upload_to_s3(self, filename, directoryName, output_format, full_path):
...
 
         s3_client.upload_file(
-            full_path, bucket, f'{directoryName}/{filename}.{output_format}')
+            full_path, bucket, object_key)
         self.log.info(
             f'File "{filename}.{output_format}" was uploaded to bucket "{bucket}" succesfully')
+        download_url = s3_client.generate_presigned_url('get_object',
+                                                    Params={'Bucket': bucket,
+                                                            'Key': object_key},
+                                                    ExpiresIn=self.__config["s3"]["download_expired_time"])
```
Comment (asafMasa) [inline src/exportImage.py:110]: set the expired flag in the DB as well

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/exportImage.py (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply src/exportImage.py:110]: TODO after prod.

File: src/exportImage.py (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply src/exportImage.py]: fixed in df899c8

**Review by asafMasa — APPROVED** [review]

**Pull request merged**